exercises/G-conditionals-2/exercise-1.js

```diff
@@ -6,7 +6,14 @@
   - if number is more or equal to zero, return the word "positive"
 */
 
-function negativeOrPositive(number) {}
+function negativeOrPositive(number) {
+  if (number < 0) {
+    return "negative";
+  }
+  else if (number >= 0) {
+    return "positive"
+  }
+}
 
 /* 
   DO NOT EDIT BELOW THIS LINE
```

Comment:
I compared your code with mine, I did exactly same , but what I find out  and learn from reviewing your code is that when something is not true returning false then for false , we don't need to put new condition. so I changed mine to just else and it works.